pymeasure/instruments/racal/racal1992.py

```diff
@@ -227,10 +235,10 @@ def read(self):
         return self.read_bytes(21).decode('utf-8')
 
     def write(self, s):
-        # Space added in front of all commands that are sent to the
-        # instrument to work around weird model issue. It shouldn't
-        # be needed on almost all devices, but it also doesn't hurt. And it
-        # fixes a real issue that's seen on a few device.
+        """Space added in front of all commands that are sent to the
+        instrument to work around weird model issue. It shouldn't
+        be needed on almost all devices, but it also doesn't hurt. And it
+        fixes a real issue that's seen on a few device."""
```

Comment:
Also here, a first, imperative clause:
```suggestion
        """Add a space in front of all commands that are sent to the
        instrument to work around weird model issue.

        It shouldn't be needed on almost all devices, but it also doesn't hurt.
        And it fixes a real issue that's seen on a few devices."""
```